README.md

```diff
@@ -51,8 +51,10 @@ java -Dloader.path="PATH-TO-ADDITIONAL-PLUGINGS/custom-plugins.jar" \
 ```
 
 The plugin library can be swapped with any third party access-checker as
-described in the [plugins](plugins) directory. Note Spring Boot is not a
-requirement for using the access-proxy; we just use it to simplify the
+described in the [plugins](plugins) directory and [the
+wiki](https://github.com/google/fhir-gateway/wiki/Understanding-access-checker-plugins).
+
+Note: Spring Boot is not a requirement for using the access-proxy; we just use it to simplify the
```

Comment:
```suggestion
Note: Spring Boot is not a requirement for using the access-gateway; we just use it to simplify the
```

Reply:
It is probably worth clarifying that we use "proxy" and "gateway" interchangeably here.

Reply:
Added a note at the top, reworded a few cases, and updated some links from proxy to gateway.